5-WebApp-AuthZ/5-1-Roles/Controllers/HomeController.cs

```diff
@@ -19,11 +22,13 @@ public class HomeController : Controller
     {
         private readonly ITokenAcquisition tokenAcquisition;
         private readonly WebOptions webOptions;
+        private readonly MicrosoftIdentityConsentAndConditionalAccessHandler _consentHandler;
 
-        public HomeController(ITokenAcquisition tokenAcquisition, IOptions<WebOptions> webOptionValue)
+        public HomeController(ITokenAcquisition tokenAcquisition, IOptions<WebOptions> webOptionValue, MicrosoftIdentityConsentAndConditionalAccessHandler consentHandler)
```

Comment:
>MicrosoftIdentityConsentAndConditionalAccessHandler [](http://example.com/codeflow?start=103&length=51)

Once you do AddMicrosoftGraph() in the startup.cs, the GraphServiceClient is injected here in the constructor, 
see [https://github.com/Azure-Samples/active-directory-aspnetcore-webapp-openidconnect-v2/blob/master/2-WebApp-graph-user/2-1-Call-MSGraph/Controllers/HomeController.cs](https://github.com/Azure-Samples/active-directory-aspnetcore-webapp-openidconnect-v2/blob/master/2-WebApp-graph-user/2-1-Call-MSGraph/Controllers/HomeController.cs)